Documentation/subcommands/api-service.md

```diff
@@ -17,3 +17,21 @@ Specify the address `0.0.0.0` to listen on all interfaces.
 
 The interfaces are defined in the [protobuf here](../../api/v1alpha/api.proto).
 Here is a small [Go program](../../api/v1alpha/client_example.go) that illustrates how to use the API service.
+
+## Options
+
+| Flag | Default | Options | Description |
+| --- | --- | --- | --- |
+| `--listen` |  `localhost:15441` | An address to listen on | Address to listen on client API requests |
+
+## Global options
```

Comment:
I agree that it reduces the possibility of mistakes and makes changes easier. However, it's less convenient for the user and I don't think we'll have many changes (and if we do: regexp!).

If only we had Markdown embedding...